### PR TITLE
Alacritty: update to 0.3.3

### PR DESCRIPTION
**Pull request by gabrielsimoes**

### CODE REVIEW

**Comment by jnbr** [comment]
according to the changelog, it doesn't depend on xclip anymore

**Comment by gabrielsimoes** [comment]
Thanks, I fixed it.

**Comment by jnbr** [comment]
If you can, please try to build this package and make sure it works.
You need at least `python3` in `hostmakedepends` and `libxcp-devel` in `makedepends`.
Also, install fails and needs to be fixed.

When you're done, squash all commits to a single one with commit message `alacritty: update to 0.3.3.`.

**Comment by gabrielsimoes** [comment]
Thanks for the help, it should be working now.

**Comment by jnbr** [comment]
@thypon are you ok with maintainer change?

**Comment by thypon** [comment]
I still use this package daily, I would prefer to maintain the ownership, or otherwise if some other core maintainer could maintain that I'm ok: the disappearance rate is high for external contributors. Feel free to ask for other packages too.

File: srcpkgs/alacritty/template [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
 build_style=cargo
-hostmakedepends="pkg-config"
-makedepends="freetype-devel fontconfig-devel"
-depends="libXxf86vm xclip ncurses alacritty-terminfo-${version}_${revision}"
+hostmakedepends="pkg-config python3"
+makedepends="freetype-devel fontconfig-devel libxcb-devel"
+depends="libXxf86vm ncurses alacritty-terminfo-${version}_${revision}"
 short_desc="Cross-platform, GPU-accelerated terminal emulator"
-maintainer="Andrea Brancaleoni <abc@pompel.me>"
+maintainer="Gabriel Silva Simoes <simoes.sgabriel@gmail.com>"
```
Comment (jnbr) [inline srcpkgs/alacritty/template]: Please revert this change.

Comment (gabrielsimoes) [reply srcpkgs/alacritty/template]: Sure, no problem. Just did it.
